mandatory/2-function-creation.js

```diff
@@ -15,17 +20,31 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let conditions = typeof num === "number" && num % 2 === 0 && num <= 100;
+  // The below code is the longer solution
+  //if (conditions) {
+  //   return true;
+  // } else {
+  //   return false;
+  // }
+  return conditions;
+}
 
 /* 
 Write a function that returns a copy of the given array arr, but with the element at the given index, index removed.
 The function must NOT change the original array, arr.
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let newArray = [];
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
Since our last discussion, I found a way to do this in a simpler manner using `splice` and `slice`. We can discuss this in our meeting :)

Reply:
Sure we can discuss it. I have done it that way too.

Reply:
In your solution above, you don't actually need `slice`. Try to check what `splice` does

Reply:
Thanks done it using splice . Splice modifies the array in place hence the need for a deep copy of the array. And one thing I learnt about it is just like pop() and shift() splice () returns the item that has been removed at index. Live and learn!